.github/workflows/deploy.yml

```
@@ -31,7 +31,7 @@ jobs:
           token: ${{ secrets.GH_TOKEN }}
       - uses: actions/setup-node@v2-beta
         with:
-          node-version: '16.10.0'
+          node-version: '18.6.0'
```

Might as well use the latest I think
```suggestion
          node-version: '18.8.0'
```